pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = -p no:dash
```

Comment:
This was added to disable the dash pytest plugin, which was causing this error (https://travis-ci.org/github/holoviz/holoviews/jobs/731249907):

```
pluggy.manager.PluginValidationError: Plugin 'dash' could not be loaded: (dash-table 4.4.1 (/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages), Requirement.parse('dash-table==4.10.1'))!
```

Reply:
Would it be better to add dash-table to the testing envt so that the plugin doesn't have to be disabled?

Reply:
I did try that, but the version on the main anaconda channel is too old (`4.4.1`). We'd have to pull `4.10.*` from conda-forge, and it doesn't look like any packages are pulled from conda-forge right now.